selftests/functional/test_output.py

```diff
@@ -435,8 +435,8 @@ def test_default_enabled_plugins(self):
         self.assertGreaterEqual(len(output_lines), 6,
                                 ('Basic human interface did not produce the '
                                  'expect output. Output produced: "%s"' % output))
-        third_line = output_lines[2]
-        debug_log = third_line.split()[-1]
+        second_line = output_lines[1]
+        debug_log = second_line.split()[-1]
```

Comment:
I was going to tell that we need to revert the code of this test, but I see you just did :)